src/main/scala/org/broadinstitute/dsde/workbench/sam/util/ShadowCaster.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.dsde.workbench.sam.util
+import cats.effect._
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.duration.MILLISECONDS
+import scala.concurrent.{ExecutionContext, Future}
+
+trait ShadowCaster extends LazyLogging {
```

Comment:
I find the term "caster" to be really distracting, like it's a D&D character or something